examples/start_building/v2/go/yourFirstTransaction.go

```diff
@@ -111,7 +69,8 @@ func main() {
 	}
 	fromAddr := myAddress
 	toAddr := "GD64YIY3TWGDMCNPP553DZPPR6LDUSFQOIJVFDPPXWEG3FVOJCCDBBHU5A"
-	var amount uint64 = 9000000
+	// close to dispenser
```

Comment:
we dont do this anymore, can you please remove

Reply:
ya, caught that too - removed the comment